ws-server/src/server/module.rs

```diff
@@ -0,0 +1,161 @@
+use super::*;
+
+pub struct RpcModule {
```

Comment:
maybe derive Default?

```suggestion
#[derive(Default)]
pub struct RpcModule {
```